velox/expression/tests/FunctionCallToSpecialFormTest.cpp

```diff
@@ -21,6 +21,8 @@
 #include "velox/expression/ConjunctExpr.h"
 #include "velox/expression/ConstantExpr.h"
 #include "velox/expression/FunctionCallToSpecialForm.h"
+#include "velox/expression/RegisterSpecialForm.h"
+#include "velox/expression/SpecialFormRegistry.h"
```

Comment:
Do you still need this include?

Reply:
Yes, because it uses `isFunctionCallToSpecialFormRegistered` and `registerFunctionCallToSpecialForms()`